screens/Home/HomeScreen.tsx

```diff
@@ -1,78 +1,89 @@
 import {
   TouchableOpacity,
+  SafeAreaView,
   StyleSheet,
   Text,
   View,
   ScrollView,
+  Image,
+
 } from "react-native";
-import React from "react";
+import Category from "./Category";
+import React, { useState } from "react";
 import { useNavigation } from "@react-navigation/core";
+import { useProductsQuery } from "../../services/productsApi";
+import { Product } from "../../models/products.model";
+
 
 const HomeScreen = () => {
+  const { data: products, error, isLoading, isFetching, isSuccess } = useProductsQuery()
+
   const navigation = useNavigation();
 
-  return (
-    <View style={{ flex: 1, backgroundColor: "#dcb688" }}>
+  const sampleFourProducts = products?.slice(3, -5)
+  const productsNotOnscreen = sampleFourProducts?.splice(2, 8)
```

Comment:
Excellent!  Much more descriptive.

Reply:
Thanks so much  Kendrick for taking the time reviewing and for all your feedback !